packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -2019,6 +2053,16 @@ export const startMultipartUpload = (
         .split('')
         .map(bit => bit === '1')
       status.clientSidePartsState = clientSidePartsState
+      let progress = 0
```

Comment:
progress is reported by number of file parts that we have processed, so the fidelity has been reduced (used to report on individual bytes in Synapse.org).  But for any file (large with multiple parts, or smaller than a single part) where the user has a reasonably good connection, the UX will be ok.